examples/multiranger/wall_following/wall_following.py

```diff
@@ -94,192 +129,250 @@ def wrap_to_pi(self, number):
         else:
             return number
 
-    def command_turn(self, ref_rate):
-        cmd_vel_x = 0.0
-        cmd_ang_w = self.direction * ref_rate
-        return cmd_vel_x, cmd_ang_w
+    # Command functions
+    def command_turn(self, reference_rate):
+        """
+        Command the Crazyflie to turn around its yaw axis
+
+        reference_rate and rate_yaw is defined in rad/s
+        velocity_x is defined in m/s
+        """
+        velocity_x = 0.0
+        rate_yaw = self.wall_following_direction_value * reference_rate
+        return velocity_x, rate_yaw
+
+    def command_align_corner(self, reference_rate, side_range, wanted_distance_from_corner):
+        """
+        Command the Crazyflie to align itself to the outer corner
+            and make sure it's at a certain distance from it
 
-    def command_align_corner(self, ref_rate, range, wanted_distance_from_corner):
-        if range > wanted_distance_from_corner + 0.3:
-            cmd_ang_w = self.direction * ref_rate
-            cmd_vel_y = 0
+        side_range and wanted_distance_from_corner is defined in m
+        reference_rate and rate_yaw is defined in rad/s
+        velocity_x is defined in m/s
+        """
+        if side_range > wanted_distance_from_corner + 0.3:
```

Comment:
Is this 0.3 the same as `range_lost_threshold`? If not, maybe use a constant or variable?

Reply:
Yeah I think that is probably supposed to be that variable. Changing it!